examples/material/api_interoperability_showcase.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "matched-portal",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    exec(urllib.request.urlopen('https://raw.githubusercontent.com/Exabyte-io/api-examples/dev/examples/utils/initialize_settings.py').read())
</pre><p><code>initialize_settings.py</code> just runs the <code>execute()</code> function and sets an environment variable. Loading a python file and executing its contents does not seem to be the best way. Can we not just import the execute function and run it here? This would require a small adjustment to the API client and is thus perhaps better dealt with in a follow-up task.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
This is the old approach which I haven't touched in this file as it was not listed in the table in README. I'll discuss with @timurbazhirov what to do with this example.

The current approach of getting the remote code we agreed on is listed in the first cells of the notebooks from the table, e.g.:
```py
#@title Authorization Form
ACCOUNT_ID = "ACCOUNT_ID" #@param {type:"string"}
AUTH_TOKEN = "AUTH_TOKEN" #@param {type:"string"}
MATERIALS_PROJECT_API_KEY = "MATERIALS_PROJECT_API_KEY" #@param {type:"string"}
ORGANIZATION_ID  = "ORGANIZATION_ID" #@param {type:"string"}

import os
if "COLAB_JUPYTER_IP" in os.environ:
    os.environ.update(
        dict(
            ACCOUNT_ID=ACCOUNT_ID,
            AUTH_TOKEN=AUTH_TOKEN,
            MATERIALS_PROJECT_API_KEY=MATERIALS_PROJECT_API_KEY,
            ORGANIZATION_ID=ORGANIZATION_ID,
        )
    )

    !GIT_BRANCH="bugfix/SOF-5578-WIP"; export GIT_BRANCH; curl -s "https://raw.githubusercontent.com/Exabyte-io/api-examples/${GIT_BRANCH}/scripts/env.sh" | bash
```